employees.json

```diff
@@ -247,6 +247,20 @@
 			],
 			"research":"coming soon",
 			"alias":"seiichi"
+		},
+		{
+			"name":"Daisuke Matsuura",
+			"degrees":[
+				{
+					"degree":"Ph.D.",
+					"subject":"Mechanical Engineering",
+					"institution":"Tokyo Institute of Technology",
+					"year":2008
+				}
+			],
+			"research":"Extendable continuum robot arm",
+			"alias":"dmatsuura"

```

Comment:
Don't forget to add a comma at the end of this line. Otherwise, the JSON file won't be readable.

Reply:
I added the comma in line 262. Is  the file OK now?